Documentation/learning/api_guarantees.md

```diff
@@ -31,36 +31,24 @@ An etcd operation that modifies the key value store is assigned a single increas
 
 All API requests are atomic; an operation either completes entirely or not at all. For watch requests, all events generated by one operation will be in one watch response. Watch never observes partial events for a single operation.
 
-#### Consistency
-
-All API calls ensure [sequential consistency][seq_consistency], the strongest consistency guarantee available from distributed systems. No matter which etcd member server a client makes requests to, a client reads the same events in the same order. If two members complete the same number of operations, the state of the two members is consistent.
-
-For watch operations, etcd guarantees to return the same value for the same key across all members for the same revision. For range operations, etcd has a similar guarantee for [linearized][Linearizability] access; serialized access may be behind the quorum state, so that the later revision is not yet available.
-
-As with all distributed systems, it is impossible for etcd to ensure [strict consistency][strict_consistency]. etcd does not guarantee that it will return to a read the “most recent” value (as measured by a wall clock when a request is completed) available on any cluster member.
-
-#### Isolation
-
-etcd ensures [serializable isolation][serializable_isolation], which is the highest isolation level available in distributed systems. Read operations will never observe any intermediate data.
-
 #### Durability
 
 Any completed operations are durable. All accessible data is also durable data. A read will never return data that has not been made durable.
 
-#### Linearizability
+#### Isolation level and consistency of replicas
+
+etcd ensures [strict serializability][strict_serializability], which is the strongest isolation guarantee of distributed transactional database systems. Read operations will never observe any intermediate data.
```

Comment:
Technically strict-1SR isn't *the* strongest consistency property, at least by implication--the empty set is the strongest model under that formalism. There are other implemented properties, like strict variants of conflict serializable, which are technically stronger, but I don't think the distinction is all that meaningful for users. I usually say it's "one of the strongest" to hedge my bets. :)

This is almost certainly fine, just noting it in case anyone wants to nitpick later.

Reply:
Ah, now that I'm reading this in context, you might also want to note here that watch ops and "serializable" reads aren't strict serializable, but everything else is.